skillmap/src/components/SkillGraphContainer.tsx

```diff
@@ -67,6 +72,14 @@ export class SkillGraphContainerImpl extends React.Component<SkillGraphContainer
             }
         }
 
+        if (darkBackgroundColors.includes(backgroundColor) ) {
```

Comment:
Could this just be using `strokeColor` prop instead of narrowing to a list of currently used background colors? typically I would expect that to be properly contrasting with `backgroundColor` or else we'd likely have other coloring issues.

(and if it needs special casing for high contrast mode I'd think a css rule for `.high-contrast .skill-graph-background img` would handle it for you)

Reply:
So, `strokeColor` can work, but it won't be a fully inclusive solution. This is because `strokeColor` would be accessed through the app's theme prop which is of type `SkillGraphTheme` ([here](https://github.com/microsoft/pxt/blob/98d18583829a877a7e2ed6373fac888d0a8a1acc/skillmap/src/lib/skillMap.d.ts#L143)), making `strokeColor` the color around the skill map graph nodes. The background images create enough separation from the background color so the `strokeColor` is often just black. If I changed the condition from `if (darkbackgroundColors.includes(backgroundColor))` to something like `if (backgroundColor === strokeColor)` that would solve the problem with strictly black backgrounds but will we will have an accessibility color contrast issue on, currently, just the Shark Splash skillmap. If any more skillmaps are created with a darker background where the alt text shouldn't be black, then we'd have to re-think something. 

So, the `strokeColor` solution should be fine for now and would be more efficient than the list look-up, and we would have to be okay with not having _all_ our skillmaps have alt text that will pass accessibility for color contrast.